### ISSUE TITLE
Bug: cannot stat '~/.sdkman/tmp/stage/contrib/*': No such file or directory

### ISSUE THREAD

**Issue by gorauskas** [sdkman/sdkman-cli#866]
**Bug report**
This is a new sdkman installation on new machine running Ubuntu 20.04 with Bash 5.0.17 

When I run `curl -s "https://get.sdkman.io" | bash` I get the error: 

    mv: cannot stat '~/.sdkman/tmp/stage/contrib/*': No such file or directory

immediately after the script echos the message `Install contributed software...`

After the installation finishes, I am able to invoke the `sdk` command.

**To reproduce**
I simply run `curl -s "https://get.sdkman.io" | bash` on my system

**System info**

```
> uname -a
Linux westminster 5.8.0-41-generic #46~20.04.1-Ubuntu SMP Mon Jan 18 17:52:23 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
> lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

```
> bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

```
> sdk version

SDKMAN 5.10.0+617
```

**Comment by marc0der** [comment]
It's a new feature that is rolling out (bash/zsh completion). Even though you will see this error, it doesn't break anything and will go away as soon as the contrib folder is in the zip distribution.

**Comment by marc0der** [comment]
This has now been resolved with the release of 5.11.0. Thanks again for reporting!
